app/modules/progress/models.py

```diff
@@ -149,6 +149,13 @@ def inactive(self):
 
     @property
     def ahead(self):
+        try:
+            return self._attempt_ahead()
+        except Exception as ex:
+            log.warning(f'failed getting ahead on {self} due to {ex}')
```

Comment:
I'm assuming that effectively ignoring this error is safe in all cases as you have done so as a bugfix.